backend/internal/service/handler/spotify/fetch_albums.go

```diff
@@ -0,0 +1,105 @@
+package spotify
+
+import (
+	"platnm/internal/models"
+	"platnm/internal/service/ctxt"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/zmb3/spotify/v2"
+)
+
+type AlbumArtist struct {
+	AlbumID  int
+	ArtistID int
+}
+
+type AddedContent struct {
+	Albums  []*models.Album
+	Artists []*models.Artist
+}
+
+func (h *SpotifyHandler) GetNewReleases(c *fiber.Ctx) error {
+	newReleases, err := fetchNewReleasesFromSpotify(c)
+	if err != nil {
+		return err
+	}
+
+	addedContent := &AddedContent{}
+	for _, album := range newReleases.Albums {
```

Comment:
use concurrency here. would involve a sync.WaitGroup, go routines, & channels. would resemble the fan out fan in pattern, [see](https://www.educative.io/answers/what-is-the-fan-out-fan-in-pattern-in-golang). we can walk through this at the TLs meeting on Wed as well